privacy_bot/mining/fetcher.py

```diff
@@ -54,6 +29,41 @@ def page_has_loaded():
 }
 
 
+async def async_fetch(session, url, timeout=10):
+    try:
+        with aiohttp.Timeout(timeout, loop=session.loop):
+            ext = tldextract.extract(url)
+            suffix = ext.suffix
+            headers = {}
+            headers['User-agent'] = 'Mozilla/5.0 (Windows NT 10.0; WOW64; rv:50.0) Gecko/20100101 Firefox/52.0'
```

Comment:
Let's  use the global variable `USERAGENT` since we use it later in the headless fetcher as well.